evil-space.el

```diff
@@ -58,6 +58,19 @@
     "Key that triggers the repeat motion in reverse direction."
     :group 'evil-space))
 
+(defun evil-space-lookup-key (key &optional keymap)
+  "Normalize KEY into a function."
+  (or (if (functionp key) key)
```

Comment:
(Oh! Didn't notice this note!) Good idea, I'll change that.

Also, I am doing some additional testing, so I may take another day or two.